cli/running/running.go

```diff
@@ -659,6 +659,50 @@ func Stop(run *InstanceCtx) error {
 	return nil
 }
 
+// Kill the Instance.
+func Kill(run *InstanceCtx) error {
+	pid, err := process_utils.KillProcess(run.PIDFile)
+	if err != nil {
+		return err
+	}
+
+	// remove PID files, because due to
+	// SIGKILL watchdog can't cleanup itself
+	cleanup(run)
+
+	// tarantool 1.10 does not have a trigger on terminate a process.
+	// So the socket will be closed automatically on termination and
```

Comment:
Will not be closed?

Reply:
fixed

Reply:
What has been fixed? I'm just confused by termination trigger. Do >1.10 versions have this trigger, which closes/removes the socket? If <=1.0 version dot not have such trigger, why "will be closed" and not "will not be closed"?